src/classes/RD2_EntryFormController.cls

```diff
@@ -56,24 +56,32 @@ public with sharing class RD2_EntryFormController {
         'SystemModStamp'
     };
 
+
     /**
     * @description  Gather the org, object and environment setting for Recurring Donation Entry/Edit Form
     * @param parentId The parentId of the new RD
     * @return Map<String, Object>
     */
-    @AuraEnabled(Cacheable=true)
+    @AuraEnabled(cacheable=true)
     public static Map<String, Object> getRecurringSettings(Id parentId) {
         String parentSObjectType = (parentId == null)
             ? null
             : parentId.getSobjectType().getDescribe().getName();
 
+        // Checks whether the Elevate credit card can be displayed
+        // on the form to enter a new Recurring Donation in NPSP and the related commitment in Elevate.
+        // The RD2 controller should be using isIntegrationEnabled() only when GA.
+        Boolean isElevateCustomer = RD2_ElevateIntegrationService.isIntegrationEnabled()
```

Comment:
The `PilotEnabled` feature parameter and its override on a sandbox will be used to enable Elevate credit card widget *in addition* to having the org connected to Elevate. When functionality becomes GA, the `PilotEnabled` check can be removed.